python-sdk/src/astro/databases/snowflake.py

```diff
@@ -601,7 +609,13 @@ def load_file_to_table_natively(
         """
         native_support_kwargs = native_support_kwargs or {}
         storage_integration = native_support_kwargs.get("storage_integration")
-        stage = self.create_stage(file=source_file, storage_integration=storage_integration)
+        if not load_options:
+            load_options = SnowflakeLoadOptions()
+        if not isinstance(load_options, SnowflakeLoadOptions):
+            raise ValueError("Error: Requires a SnowflakeLoadOptions")
+        stage = self.create_stage(
+            file=source_file, storage_integration=storage_integration, load_options=load_options
+        )
 
         table_name = self.get_table_qualified_name(target_table)
         file_path = os.path.basename(source_file.path) or ""
```

Comment:
Can we move this block and the below line to construct the `sql_statement` into `_copy_into_table_from_stage` method. Because otherwise the `_copy_into_table_from_stage` is just executing a `sql_statement` and checking for exception; it does not give a hint of what the method name means, the `sql_statement` is abstract to the method

Reply:
done